include/ignition/physics/ConstructEmpty.hh

```diff
@@ -73,6 +73,30 @@ class ConstructEmptyModelFeature : public virtual Feature
   };
 };
 
+/////////////////////////////////////////////////
+/// \brief This feature constructs an empty nested model and return its pointer
+/// from the given world.
+class ConstructEmptyNestedModelFeature : public virtual Feature
+{
+  public: template <typename PolicyT, typename FeaturesT>
+  class Model : public virtual Feature::Model<PolicyT, FeaturesT>
+  {
+    public: using ModelPtrType = ModelPtr<PolicyT, FeaturesT>;
+
+    /// \brief Construct an empty model and attach a given name to it.
+    /// \return
+    ///   The ModelPtrType of the constructed model.
```

Comment:
> It looks like all the other method documentation in this file has a new line between \return and the corresponding information, is there a reason for this formatting?

I don't know what the reason is, but I was being consistent with the style in the file. Doxygen works either way. I think it might be a preference thing.

> Also, I noticed documentation for other methods in this file omits param[in], but I'm not sure why. Is there any particular reason for leaving out method parameter documentation?

No, I think that's just an oversight. I'll add those in.

Reply:
Added in https://github.com/ignitionrobotics/ign-physics/pull/228/commits/b836a0aa0fc7e9407e9f9d569491d3b9d32d9f82